services/src/main/java/se/inera/intyg/common/services/texts/DefaultCertificateTextProvider.java

```diff
@@ -23,9 +23,11 @@
 public final class DefaultCertificateTextProvider implements CertificateTextProvider {
 
     private final IntygTexts intygTexts;
-    private static final String ANGULAR_LAUNCH_ICON = "<i class=\"material-icons md-18\">launch</i>";
+    private static final String ANGULAR_LAUNCH_ICON =
```

Comment:
Suggest a different naming of the constants now when its not about replacing the icon, but instead replacing the whole link.